modules/Emails/Email.php

```diff
@@ -4364,16 +4364,16 @@ private function sendOptInEmailToEmailAddressById($id)
             $emailAddresses = BeanFactory::getBean('EmailAddresses');
             $emailAddress = $emailAddresses->retrieve($id);
 
-            if ($emailAddress->confirm_opt_in != 'confirmed-opt-in' && empty($emailAddress->opt_in_email_created)) {
```

Comment:
`opt_in_email_created` renamed but still used in test: SugarEmailAddressTest.php:844, perhaps  this test will failing.